mobileApp/resources/copy_keys.sh

```diff
@@ -1,6 +1,7 @@
-#!/usr/bin/bash env
+#!/usr/bin/env bash -e
```

Comment:
Oh I didn't realize you could pass in the `-e` on the hashbang line. This works?